SimGeneral/MixingModule/python/SiPixelSimParameters_cfi.py

```diff
@@ -49,6 +49,8 @@ def _modifyPixelDigitizerForRun3( digitizer ):
     SiPixelQualityLabel = cms.string(''),
     KillBadFEDChannels = cms.bool(False),
     UseReweighting = cms.bool(False),
+    applyLateReweighting = cms.untracked.bool(False),
+    store_SimHitEntryExitPoints = cms.untracked.bool(False),
```

Comment:
```suggestion
    applyLateReweighting = cms.bool(False),
    store_SimHitEntryExitPoints = cms.bool(False),
```

Reply:
This does not work, because the code is expecting an untracked parameter for applyLateReweighting.

Reply:
Ah, right. What is also needed are corresponding changes in `SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizer.cc`. I now proposed the code modifications there as well.